firedrake/utility_meshes.py

```diff
@@ -541,9 +543,11 @@ def RectangleMesh(
 
     :arg nx: The number of cells in the x direction
     :arg ny: The number of cells in the y direction
-    :arg Lx: The extent in the x direction
-    :arg Ly: The extent in the y direction
-    :kwarg quadrilateral: (optional), creates quadrilateral mesh.
+    :arg Lx: The extent in the x direction from the origin
+    :arg Ly: The extent in the y direction from the origin
+    :arg originX: The X coordintes of the origin, deafult 0.
+    :arg originY: The Y coordintes of the origin, deafult 0.
```

Comment:
```suggestion
    :arg originX: The X coordinates of the origin, default 0.
    :arg originY: The Y coordinates of the origin, default 0.
```

Reply:
@ReubenHill thank you so much, my dyslexia was acting up!

Reply:
@ReubenHill Should I remove the default value also here?